app/services/add_ons/apply_taxes_service.rb

```diff
@@ -20,7 +20,7 @@ def call
       ).destroy_all
 
       result.applied_taxes = tax_codes.map do |tax_code|
-        add_on.applied_taxes.find_or_create_by!(tax: taxes.find_by(code: tax_code))
+        add_on.applied_taxes.find_or_create_by!(tax: taxes.find_by(code: tax_code), organization: add_on.organization)
```

Comment:
The `find_or_create_by` my lead to an error if the organization_id is not yet backfilled.

To avoid this you could do:
```suggestion
        add_on.applied_taxes.create_with(organization_id: add_on.organization_id).find_or_create_by!(tax: taxes.find_by(code: tax_code))
```